codesystems/CodeSystem-England-PrescriptionOrderingParameters.xml

```diff
@@ -0,0 +1,33 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="England-PrescriptionOrderingParameters" />
+  <url value="https://fhir.nhs.uk/England/CodeSystem/England-PrescriptionOrderingParameters" />
+  <version value="1.0.0" />
+  <name value="EnglandPrescriptionOrderingParameters" />
+  <title value="England Prescription Ordering Parameters" />
+  <status value="draft" />
+  <date value="2023-11-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="A code set to define parameters provided when managing patients' prescription" />
```

Comment:
Suggest description to be 'A set of codes that define parameters provided when managing a patient's prescription'. (a change to the beginning an also updated patients' to patient's)

Reply:
happy with that if it's more consistent with other assets.